config/migrations/2024/bestuurseenheden/20240919101332-add-new-welzijnsvereniging-sint-anna/20240919101332-add-new-organisation-sint-anna.sparql

```diff
@@ -0,0 +1,54 @@
+      <http://data.lblod.info/id/bestuursorganen/40d628b0-6c5d-11ef-8f29-2728b827bd59> rdf:type	<http://data.vlaanderen.be/ns/besluit#Bestuursorgaan> ;
+        <http://mu.semte.ch/vocabularies/core/uuid>	"40d628b0-6c5d-11ef-8f29-2728b827bd59" ;
+        <http://data.vlaanderen.be/ns/besluit#classificatie>	<http://data.vlaanderen.be/id/concept/BestuursorgaanClassificatieCode/41caf7e6-b040-4720-9cc2-a96cfffed5b4> ;
+        <http://data.vlaanderen.be/ns/besluit#bestuurt>	<http://data.lblod.info/id/bestuurseenheden/66DB132B6ABCA6E6BF2DBBED> .
+
+      <http://data.lblod.info/id/bestuursorganen/40bfba80-6c5d-11ef-8f29-2728b827bd59> rdf:type	<http://data.vlaanderen.be/ns/besluit#Bestuursorgaan> ;
+	      <http://mu.semte.ch/vocabularies/core/uuid>	"40bfba80-6c5d-11ef-8f29-2728b827bd59" ;
+	      <http://data.vlaanderen.be/ns/besluit#classificatie>	<http://data.vlaanderen.be/id/concept/BestuursorgaanClassificatieCode/013cc838-173a-4657-b1ae-b00c048df943> ;
+	      <http://data.vlaanderen.be/ns/besluit#bestuurt>	<http://data.lblod.info/id/bestuurseenheden/66DB132B6ABCA6E6BF2DBBED> .
+
+      <http://data.lblod.info/id/bestuursorganen/40cab700-6c5d-11ef-8f29-2728b827bd59> rdf:type	<http://data.vlaanderen.be/ns/besluit#Bestuursorgaan> ;
+        <http://mu.semte.ch/vocabularies/core/uuid>	"40cab700-6c5d-11ef-8f29-2728b827bd59" ;
+        <http://data.vlaanderen.be/ns/besluit#classificatie>	<http://data.vlaanderen.be/id/concept/BestuursorgaanClassificatieCode/b52094ff-21a2-4da8-8dbe-f513365d1528> ;
+        <http://data.vlaanderen.be/ns/besluit#bestuurt>	<http://data.lblod.info/id/bestuurseenheden/66DB132B6ABCA6E6BF2DBBED> .
+
+      # Insert bestuursorganen in tijd
+      <http://data.lblod.info/id/bestuursorganen/40ec21b0-6c5d-11ef-8f29-2728b827bd59>
+        rdf:type	<http://data.vlaanderen.be/ns/besluit#Bestuursorgaan> ;
+        <http://mu.semte.ch/vocabularies/core/uuid>	"40ec21b0-6c5d-11ef-8f29-2728b827bd59" ;
+        <http://data.vlaanderen.be/ns/mandaat#bindingStart>	"2019-01-01T00:00:00Z"^^xsd:dateTime ;
+        <https://data.vlaanderen.be/ns/generiek#isTijdspecialisatieVan>	<http://data.lblod.info/id/bestuursorganen/40d628b0-6c5d-11ef-8f29-2728b827bd59> .
+
+      <http://data.lblod.info/id/bestuursorganen/40e4f5c0-6c5d-11ef-8f29-2728b827bd59>
+        rdf:type	<http://data.vlaanderen.be/ns/besluit#Bestuursorgaan> ;
+        <http://mu.semte.ch/vocabularies/core/uuid>	"40e4f5c0-6c5d-11ef-8f29-2728b827bd59" ;
+        <http://data.vlaanderen.be/ns/mandaat#bindingStart>	"2019-01-01T00:00:00Z"^^xsd:dateTime ;
+        <https://data.vlaanderen.be/ns/generiek#isTijdspecialisatieVan>	<http://data.lblod.info/id/bestuursorganen/40bfba80-6c5d-11ef-8f29-2728b827bd59> .
+
+      <http://data.lblod.info/id/bestuursorganen/40e8ed60-6c5d-11ef-8f29-2728b827bd59> <http://data.vlaanderen.be/ns/mandaat#isTijdspecialisatieVan>	<http://data.lblod.info/id/bestuursorganen/40cab700-6c5d-11ef-8f29-2728b827bd59> . # No bestuurseenheid in tijd data, only uri
```

Comment:
It seems this bestuursorgaan in tijd does have triples in OP:

```ttl
<http://data.lblod.info/id/bestuursorganen/40e8ed60-6c5d-11ef-8f29-2728b827bd59>
    rdf:type <http://data.vlaanderen.be/ns/besluit#Bestuursorgaan> ;
    <http://mu.semte.ch/vocabularies/core/uuid> "40e8ed60-6c5d-11ef-8f29-2728b827bd59" ;
    <http://data.vlaanderen.be/ns/mandaat#isTijdspecialisatieVan> <http://data.lblod.info/id/bestuursorganen/40cab700-6c5d-11ef-8f29-2728b827bd59> ;
    <http://data.vlaanderen.be/ns/mandaat#bindingStart> "2019-01-01T00:00:00Z"^^xsd:dateTime .
```

Reply:
It seems the backup that I used doesnt have that data? or OP consumes that from somewhere else? I dont find this data. 

![image](https://github.com/user-attachments/assets/de7684d8-0a51-4798-aa1f-e1d986ee2b4e)

Reply:
purged database and replayed the backup I got from OP and still same results. This is from 240925T0430. If your data is older then this then they might have deleted those triples by 25/09. If its more recent then they added this data after 25/09

Reply:
Yes, I think they added this after 25/09. If I query prod directly, I can see the data.